### PR TITLE
Update ios project

### PR DESCRIPTION
**Pull request by maxammann**

### CODE REVIEW

**Comment by maxammann** [comment]
f-droid is down right now, so the badge is not shown yet

**Comment by maxammann** [comment]
I just updated deps

**Review by Schedulaar — APPROVED** [review]

File: frontend/ios/Runner.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -195,21 +195,21 @@
...
-				0B1FA81426DBD06400D4AEA7 /* maps_launcher.framework */,
-				0B1FA81626DBD06400D4AEA7 /* MTBBarcodeScanner.framework */,
-				0B1FA81826DBD06400D4AEA7 /* package_info.framework */,
-				0B1FA81A26DBD06400D4AEA7 /* path_provider.framework */,
-				0B1FA81C26DBD06400D4AEA7 /* Pods_Runner.framework */,
-				0B1FA81E26DBD06400D4AEA7 /* qr_code_scanner.framework */,
-				0B1FA82026DBD06400D4AEA7 /* Reachability.framework */,
-				0B1FA82226DBD06400D4AEA7 /* shared_preferences.framework */,
-				0B1FA82426DBD06400D4AEA7 /* url_launcher.framework */,
+				0BE935D1274FD4CA004AF690 /* connectivity_plus */,
```
Comment (klinzo) [inline frontend/ios/Runner.xcodeproj/project.pbxproj:198]: Are these changes really correct?

File: frontend/ios/Runner.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -973,7 +988,7 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = de.nrw.it.ehrensachebayern;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = "match AppStore de.nrw.it.ehrensachebayern";
+				PROVISIONING_PROFILE_SPECIFIER = "match Development de.nrw.it.ehrensachebayern";
```
Comment (klinzo) [inline frontend/ios/Runner.xcodeproj/project.pbxproj:991]: Is this correct? Same for change above?

**Review by klinzo — APPROVED** [review]

**Pull request merged**